src/pages/SitePolicy/policies/CookiePolicy.tsx

```diff
@@ -0,0 +1,148 @@
+import * as React from 'react';
+
+export const CookiePolicy = () => (
+  <div>
+    This Cookie Policy explains how we uses cookies and similar technologies to recognize you when you visit
+    our websites at https://iuscsg.dev, https://iuscsg.com. <br/>
+    It explains what these technologies are and why we use them, as well as your rights to control our use of them.
+    <br/><br/>
```

Comment:
I would handle spaces between paragraphs by separating your paragraphs into `<p>` tags.

```
<p>
    This Cookie Policy explains how we uses cookies and similar technologies to recognize you when you visit
    our websites at https://iuscsg.dev, https://iuscsg.com. <br/>
    It explains what these technologies are and why we use them, as well as your rights to control our use of them.
</p>
```

If you need more margin on the bottom of them, you can increase them using style. Apply the same pagraphs to the other policies.

Reply:
I am going to leave it the way it is for now if if becomes a problem I will fix it but it will take me almost 4 hours to do all of the formatting.